src/lib/stats.js

```diff
@@ -7,6 +7,7 @@ import { error } from "./loggers"
 const { NODE_ENV, ENABLE_METRICS, STATSD_HOST, STATSD_PORT } = config
 
 const isTest = NODE_ENV === "test"
+const isDev = NODE_ENV === "development"
```

Comment:
I don’t know if we really set this env, do you? In any case, it looks like functional-wise these end up being used the same way, so how about replacing these both with `production` and just check in all the places if this is _not_ production?

Reply:
Ah yes in staging `NODE_ENV=production` as well